src/cljs/rems/table.cljs

```diff
@@ -181,17 +181,21 @@
         ;;       (or then we could just not validate these internal row representations)
         (s/validate Rows))))
 
+(defn- normal-sort [sorting option1 option2]
+  (case (:sort-order sorting)
+    :desc (compare option2 option1)
+    :asc (compare option1 option2)))
+
```

Comment:
This is not identical to the old code. Why?

Reply:
I believed it would look cleaner this way, I might be wrong

Reply:
But you don't handle the default case here like the previous code did.